server/src/main/java/org/opensearch/search/query_group/QueryGroupServiceSettings.java

```diff
@@ -0,0 +1,256 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.search.query_group;
```

Comment:
This need not be in the search package right? We can move this to wlm?

Reply:
Agree!, this doesn't have anything search specific. @ruai0511 Can we move this class under WLM ?